examples/jsm/renderers/CSS3DRenderer.js

```diff
@@ -86,6 +86,7 @@ var CSS3DRenderer = function () {
 	domElement.appendChild( cameraElement );
 
 	var isIE = /Trident/i.test( navigator.userAgent );
+	var isSafari = !! navigator.userAgent.match( /Safari/i ) && ! navigator.userAgent.match( /Chrome/i );
```

Comment:
How about using test?
`test` is faster than `match` according to https://jsperf.com/test-vs-match-regex/18
```suggestion
	var isSafari = /Safari/.test( navigator.userAgent ) && ! /Chrome/.test( navigator.userAgent );
```

Reply:
Thanks a lot for the review! Sounds good to me, thanks for pointing it out. I got the isSafari check from the examples (is used in all examples/webgl_gpgpu_*.html files). Maybe we should change the check there as well?